functions.opengraph.php

```diff
@@ -94,6 +94,24 @@ function jetpack_og_tags() {
 			$tags['profile:first_name'] = get_the_author_meta( 'first_name', $author->ID );
 			$tags['profile:last_name']  = get_the_author_meta( 'last_name', $author->ID );
 		}
+	} elseif ( is_archive() ) {
+		$tags['og:type'] 		= 'website';
```

Comment:
Could you switch to using spaces instead of tabs here, as per WordPress' coding standards? If you [set up your code editor](https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#use-php-codesniffer-and-eslint-to-make-sure-your-code-respects-coding-standards), you should be able to get recommendations like so:

![image](https://user-images.githubusercontent.com/426388/62230846-011a3480-b3c3-11e9-8f6f-a0f5b7901c8c.png)

Reply:
Weird. I ran yarn build and I ran the listing tools including PHPCS and none of this came up. Apologies! Fixes coming.